Resources/Prototypes/Entities/Mobs/Player/silicon.yml

```diff
@@ -453,7 +453,7 @@
         name: power-cell-slot-component-slot-name-default
         startingItem: PowerCellMedium
   - type: RandomMetadata # DeltaV: give job borgs names
```

Comment:
change the comment to `# DeltaV - summary` while you're at it for the sake of my OCD

Reply:
pro I diddnt write that but sure